cpp/daal/include/algorithms/gradient_boosted_trees/gbt_classification_model.h

```diff
@@ -123,8 +123,26 @@ class DAAL_EXPORT Model : public classifier::Model
      */
     virtual size_t getNumberOfTrees() const = 0;
 
+    /**
+     * \brief Set the Prediction Bias term
+     *
+     * \param value global prediction bias
+     */
+    void setPredictionBias(double value) { _predictionBias = value; }
+
+    /**
+     * \brief Get the Prediction Bias term
+     *
+     * \return double prediction bias
+     */
+    double getPredictionBias() const { return _predictionBias; }
+
 protected:
     Model() : classifier::Model() {}
+
+private:
+    /* global bias applied to predictions*/
+    double _predictionBias;
```

Comment:
I think this is not fixed yet. The _predictionBias field and implementations of the respective set/get methods need to be moved into internal .cpp files.
Otherwise it will break the binary compatibility: the layout of the Model class had changed with the addition of _predictionBias field.

Reply:
My bad and thanks for the explanation!